docs/ceph-csi-upgrade.md

```diff
@@ -107,12 +108,15 @@ that will not be supported in the official releases. Builds from the devel
 branch can have functionality changed and even removed at any time without
 compatibility support and without prior notice.
 
-git checkout v3.3.0 tag
+**Also, we do not recommend any direct upgrades to 3.4 except from 3.3 to 3.4.**
```

Comment:
```suggestion
**Also, we recommend upgrading by only one release at a time.**
```

Reply:
And also, you can include the example as you have already done :+1:

Reply:
I will take it as a nit, if this PR needs any repin further, I will consider clubbing it along, else we can take it later.